IntelliTect.TestTools.Data/DatabaseFixture.cs

```diff
@@ -246,7 +244,8 @@ internal ContextConstructionInfo GetOrAddConstructionInfo<T>()
 
         if (_SeedComplete) return db;
 
-        _Seed?.Invoke();
+        if (seeding || _Seed is null) return db;
+        await _Seed.Invoke();
```

Comment:
Rather than passing in a flag that is only set in one place, why not simply pull this code into the `_Seed` delegate?